extensions/wikia/VisualEditorTourExperiment/scripts/VisualEditorTourExperimentInitArticleEntry.js

```diff
@@ -0,0 +1,61 @@
+require(['jquery', 'mw', 'VisualEditorTourExperimentInit', 'wikia.tracker'],
+	function ($, mw, veTourInit, tracker) {
+		'use strict';
+
+		var track = tracker.buildTrackingFunction({
+			category: 've-editing-tour',
+			trackingMethod: 'analytics'
+		});
+
+		function initEntry() {
+			var $editButton = $('#ca-ve-edit');
```

Comment:
Open question on an edge case: is there a chance a returning user without edits has set the source editor in their preferences? Probably unlikely, but worth a thought.